scripts/common.py

```diff
@@ -211,6 +211,13 @@
     '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome'
 ]
 
+GAE_PORT_FOR_LOCAL_E2E_TESTING: Final = 9001
```

Comment:
What does "local" mean here? Do we use a different port on CI? Or if the main point is that we use a different port in production (and this is what we use for local development) then it's not really about e2e tests

Reply:
Ah that's a good point, I might have missed that when suggesting the name. @Shivkant-Chauhan could you please check if we use the same port on CI? If so, let's drop the word "local".

Reply:
@U8NWXD @seanlip No we are going to use the same port on both local as well as on the CI. We can drop the LOCAL

Reply:
yep, pushed the changes changing name to `GAE_PORT_FOR_E2E_TESTING `